rslib/src/backend/search/search_node.rs

```diff
@@ -98,6 +98,15 @@ impl TryFrom<pb::SearchNode> for Node {
                         Node::Group(nodes)
                     }
                 }
+                Filter::Field(field) => Node::Search(SearchNode::SingleField {
+                    field: escape_anki_wildcards_for_search_node(&field.field_name),
+                    text: escape_anki_wildcards(&field.text),
```

Comment:
You could argue for any of these two functions, but maybe it would be more intuitive if the same was used in both cases?

Reply:
Is there a particular use case you had in mind for _for_search_node() @abdnh? If not, maybe the wildcards() option would be a better choice?